src/app/app.component.html

```diff
@@ -5,6 +5,8 @@
   <div class="container-fluid" [@routeAnimations]="prepareRoute(outlet)">
```

Comment:
what is the rule of this #outlet

Reply:
#<Name> In  Angular means  assigning html element to variable here outlet

Reply:
See also #dv in post-list html file for  grid element or list element